_ingest-pipelines/processors/ml-inference.md

```diff
@@ -44,30 +52,36 @@ The following is the syntax for the `ml-inference` processor:
-| `ignore_failure` | Boolean   | Optional | If `true` and any exceptions occur, then they are ignored and ingestion continues. Otherwise, an exception occurs and ingestion is canceled. Default is `false`.                                                                                                                                                                                                                                             |
+| Parameter                   | Data type | Required/Optional                                           | Description                                                                                                                                                                                                                                                                                                                                                                                                 |
+|:----------------------------|:----------|:------------------------------------------------------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `model_id`                  | String    | Required                                                    | The ID of the ML model to be used by the processor.                                                                                                                                                                                                                                                                                                                                                         |
+| `function_name`             | String    | Optional for remote models.<br/>Required for local models   | The function name of the ML model configured in the processor. The default value is `remote`                                                                                                                                                                                                                                                                                                                |
+| `input_map`                 | Array     | Optional                                                    | An array specifying how to map fields from the ingested document to the model's input. If no input mapping is specified, then all fields from the document are used as the model input by default. The `input_map` size indicates the number of prediction tasks. In each map, the key represents the model input field name, and the value specifies the document field used to construct the model input. |
```

Comment:
@rbhavna Have you verified this is optional for both remote and local model?

Reply:
Sorry this is required for local models. Will update the documentation